src/material-experimental/mdc-tabs/ink-bar.ts

```diff
@@ -138,6 +143,17 @@ export class MatInkBarFoundation {
     }
   }
 
+  getDestroyed() {
+    return this._destroyed;
+  }
+
```

Comment:
Instead of adding getters for all of these, we can use `readonly` properties instead which won't generate any extra runtime code. If we need to change one of them (e.g. `destroyed`), we can do something like this `(this as {destroyed: boolean}).destroyed = true`.

Reply:
So to be clear, the properties ```_inkBarContentElement```, ``` _hostElement```, and ```_getDestroyed``` should be exposed publicly but made ```readonly```?

Reply:
Exactly.